package.json

```diff
@@ -35,5 +35,8 @@
   },
   "resolutions": {
     "usehooks-ts@^2.7.2": "patch:usehooks-ts@npm:^2.7.2#./.yarn/patches/usehooks-ts-npm-2.7.2-fceffe0e43.patch"
+  },
+  "dependencies": {
+    "react-icons": "^4.10.1"
```

Comment:
This should be added to the packages/nextjs/package.json file, not here. Just move the reference and run yarn start and I but the errors will disappear. If they still show you may need to delete package-lock.json and yarn.lock and try again.